modules/php/Cards/Rules/RulePoorBonus.php

```diff
@@ -18,12 +18,12 @@ public function __construct($cardId, $uniqueId)
 
   public function immediateEffectOnPlay($player)
   {
-    // @TODO : set game state?
-    // + draw extra card if current player is Poor
+    Utils::getGame()->setGameStateValue("activePoorBonus", 1);
+    // @TODO: draw extra card if current player is Poor
```

Comment:
According to the card, even though it's tagger "instant effect", I think this one should be considered more a "free action", because it says "may". I guess for this one, most players will do it, but for the "rich bonus", maybe some player might decide to not play the extra turn.

So I think it would be better to consider those to like "free action".